### ISSUE TITLE
Requesting the presentation of evaluation results and reproducibility details.

### ISSUE THREAD

**Issue by ZY123-GOOD** [NeuSpeech/EEG-To-Text#1]
Thank you for your contribution. Could you please let me know when you'll be able to provide the final performance of this model and a detailed readme file for readers to reproduce? It's something we're really looking forward to. Thanks again.

**Comment by NeuSpeech** [comment]
hi, I uploaded new code and metrics at master branch, you can check out.

**Comment by YutoNishimura-v2** [comment]
@NeuSpeech 
Hello, thank you always for your contributions.

I would like to reproduce the results using the BART model, but I am not sure which script to use.

Could you kindly share the train and eval scripts corresponding to BART and BART w/tf?

Thank you in advance.

**Comment by YutoNishimura-v2** [comment]
@NeuSpeech 
Thank you for your prompt reply!

I apologize for completely forgetting that the dataset description is included in the original repository. Thank you for pointing that out. It seems that this information is correct.

On the other hand, it appears that there is only one example of the training configuration, so I would like to confirm this. In the original repository, there is the following:

```
python3 train_decoding.py --model_name BrainTranslator \
    --task_name task1_task2_taskNRv2 \
    --one_step \
    --pretrained \
    --not_load_step1_checkpoint \
    --num_epoch_step1 20 \
    --num_epoch_step2 30 \
    -lr1 0.00005 \
    -lr2 0.0000005 \
    -b 32 \
    -s ./checkpoints/decoding \
    -cuda cuda:0
```

This is for training with EEG, and the following from NeuSpeech's repository:

```
CUDA_VISIBLE_DEVICES=0 python3 train_decoding.py --model_name BrainTranslator \
    --task_name task1_task2_task3 \
    --one_step \
    --pretrained \
    --not_load_step1_checkpoint \
    --num_epoch_step1 20 \
    --num_epoch_step2 30 \
    --train_input noise \
    -lr1 0.00002 \
    -lr2 0.00002 \
    -b 32 \
    -s ./checkpoints/decoding
```

Is this for noise training? 
It seems that the **learning rates are slightly different**, though.

Also, regarding the ZuCo2 dataset, I believe it contains quite a lot of "None" values during the preprocessing stage. Is that correct?

From what I’ve observed, it seems that training with only ZuCo1 might be a better option. I would like to hear your thoughts on this.

**Comment by NeuSpeech** [comment]
maybe we changed something unintentionally, you can run it and see what happens

**Comment by YutoNishimura-v2** [comment]
@NeuSpeech 
However, this replication is unique in that the goal is to confirm that it 'doesn't work,' making it difficult to determine whether the observed results are as intended, even after running the experiment and checking the outcomes.

It would be greatly appreciated if you could simply share the actual run scripts. 
(As I mentioned before, this research is aimed at pointing out the mistakes of the pioneers, so the reproducibility must be precise.)"

**Comment by girlsending0** [comment]
@YutoNishimura-v2 

Thank you for your detailed check.

Our main experimental result is that the result values of noise input and EEG input are very similar. This is clearly the same for any running point. We present a tf code for it, and provide a code for comparing noise input values. The reason why the learning points are different is that we have learned about 3 models. Anyway, we have to drop the loss for convergence to text, and the running point that satisfies the convergence of all 3 models was different from that of the original author. But the result is the same. It is easy for you to test.

**Comment by YutoNishimura-v2** [comment]
@girlsending0 

Thank you for your reply.

Yes, I understand that part. I am simply seeking full reproducibility as a general research principle. In writing my paper, I would like to run your experiments, which are prior research, under the same settings.

It has been pointed out in other issues that differences may arise due to SEED. I believe that using the same configuration is important to prevent such mistakes from occurring.

Honestly, I’m not looking for this kind of discussion. 
I just want to finish this by having you share the scripts for the two settings of the BART model, so is there any reason why this cannot be done? 
It might be an impertinent request, but I ask for your cooperation to ensure fair research.

**Comment by YutoNishimura-v2** [comment]
Thank you for your cooperation. In the future, sharing the scripts in the README, as done in the NeuSpeech repository, would be more helpful for other researchers.

Once again, thank you for your cooperation.
